layouts/partials/header.html

```diff
@@ -95,7 +95,7 @@
 																<a href="{{ .URL }}">
 																	{{ .Name }}
 																</a>
-																<ul class="sub-menu">
+																<ul class="sub-menu menu-pos-invert">
 																	{{ range .Children }}
```

Comment:
We can see here the change because is in the right,

```html
<ul class="sub-menu menu-pos-invert">
```
the added class is **menu-pos-invert**